mkdocs.yml

```
@@ -944,7 +944,7 @@ extra:
   latest_tag_4_6: '4.6.19'
 
   symfony_doc: 'https://symfony.com/doc/5.x'
-  user_doc: 'https://doc.ibexa.co/projects/userguide/en/master'
+  user_doc: 'https://doc.ibexa.co/projects/userguide/en/latest'
```

Since we're focusing on version numbers, `master` is no more, and `latest` is just an alias, I would propose this:
```suggestion
  user_doc: 'https://doc.ibexa.co/projects/userguide/en/5.0'
```